src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/enums/Need.kt

```diff
@@ -8,5 +8,7 @@ enum class Need(val weighting: Int) {
   DRUG_MISUSE(1),
   ALCOHOL_MISUSE(1),
   THINKING_AND_BEHAVIOUR(2),
-  ATTITUDES(2)
+  ATTITUDES(2),
+  FINANCIAL_MANAGEMENT_AND_INCOME(0),
+  EMOTIONAL_WELL_BEING(0)
 }
```

Comment:
This is fine, if there are many more, we should change the client to return String and try to parse the results into a  Need in the service layer and then filter the null ones out like we do for other calls. This will  break if there are new Needs added.